CMakeLists.txt

```diff
@@ -132,12 +132,12 @@ if (USE_MFEM)
     dg_advection_global_rom
     dg_advection_local_rom_matrix_interp
     mixed_nonlinear_diffusion
+    nonlinear_elasticity_global_rom
     linear_elasticity_global_rom
     dg_advection
     nonlinear_elasticity
     heat_conduction
     parametric_heat_conduction
-    de_parametric_heat_conduction_greedy
```

Comment:
It looks like this was removed by mistake.

Reply:
Yes that is most likely!